.github/workflows/c-cpp.yml

```diff
@@ -25,3 +25,41 @@ jobs:
         run: meson compile -C build
       - name: dist
         run: meson dist -C build
+
+  build-windows:
+
+    runs-on: windows-latest
+
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    steps:
+
+      - uses: msys2/setup-msys2@v2
+      - name: install deps
+        run: |
+          pacman -S --noconfirm \
+                    git cmake \
```

Comment:
`cmake`?

Reply:
cmake is needed to detect some features, I think libproxy (?) but I'll have to double check. Either way it doesn't work, see the build logs:

```
Found CMake: D:\a\_temp\msys64\usr\bin/cmake.EXE (3.22.0)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Run-time dependency mrss found: NO (tried pkgconfig)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Run-time dependency geoip found: NO (tried pkgconfig)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Run-time dependency libproxy found: NO (tried pkgconfig)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
```

I don't know how to fix this.

Reply:
From my builds on linux:

```
Run-time dependency mrss found: YES 0.19.2
Run-time dependency geoip found: YES 1.6.12
Found CMake: /usr/bin/cmake (3.22.1)
Run-time dependency libproxy found: YES 0.4.17
Run-time dependency appindicator3-0.1 found: YES 12.10.0
```

So libproxy is the only one that needs cmake, the rest is pkg-config.

Reply:
Oh you just have the wrong name for pkg-config.

Its `libproxy-1.0` (https://github.com/libproxy/libproxy/blob/master/libproxy/libproxy-1.0.pc.in)

Reply:
whoops